File: app/src/main/java/com/example/truckdriver_v02/data/vehicle/TrailerModel.java
package com.example.truckdriver_v02.data.vehicle;

public class TrailerModel {
    private int TrailerTypeId;
    private String TrailerTypeName;

    public TrailerModel(int trailerTypeId, String trailerTypeName) {
        TrailerTypeId = trailerTypeId;
        TrailerTypeName = trailerTypeName;
    }

    public TrailerModel(String trailerTypeName) {
        TrailerTypeName = trailerTypeName;
    }

    public int getTrailerTypeId() {
        return TrailerTypeId;
    }

    public void setTrailerTypeId(int trailerTypeId) {
        TrailerTypeId = trailerTypeId;
    }

    public String getTrailerTypeName() {
        return TrailerTypeName;
    }

    public void setTrailerTypeName(String trailerTypeName) {
        TrailerTypeName = trailerTypeName;
    }
}

File: app/src/main/java/com/example/truckdriver_v02/data/vehicle/TrailerModel.kt
package com.example.truckdriver_v02.data.vehicle

class TrailerModel {
    var trailerTypeId: Int = 0
    var trailerTypeName: String

    constructor(trailerTypeId: Int, trailerTypeName: String) {
        this.trailerTypeId = trailerTypeId
        this.trailerTypeName = trailerTypeName
    }

    constructor(trailerTypeName: String) {
        this.trailerTypeName = trailerTypeName
    }
}
